Contest_API.md

```diff
@@ -1493,24 +1519,28 @@ Request data:
 
 Request:
 
-` PATCH https://example.com/api/contests/wf14/awards`
+` PUT https://example.com/api/contests/wf14/awards/best-costume`
 
 Request data:
 
 ```json
-{"id":"best-costume","citation":"Best team costumes"}
+{"citation":"Best team costuems","team_ids":["24"]}
```

Comment:
Nitpick: should probably be `costumes`.

Reply:
I was trying to make it look like they misspelled costumes and then fixed it, but wasn't obvious enough. New commit changes it to costumes and then renames citation to cosplay, hopefully this is better.